host/sgx/create.c

```diff
@@ -704,6 +708,17 @@ oe_result_t oe_create_enclave(
     /* Setup logging configuration */
     oe_log_enclave_init(enclave);
 
+    /* Peform a two-way ping with the enclave. */
```

Comment:
Probably should comment that this is a mock test, and will be removed in a follow up PR.

Reply:
Oh, good idea. Resolved.